### PR TITLE
support for spark ingestion by origin

### PR DESCRIPTION
**Pull request by khatton-ft**

### CODE REVIEW

File: service/message_handler_test.go [hunk 1]
```diff
@@ -184,6 +184,25 @@ func TestHandleWriteMessage(t *testing.T) {
 	concordanceApiMock.AssertExpectations(t)
 }
 
+func TestHandleWriteMessageSparkHeader(t *testing.T) {
+	assert := assert.New(t)
+
+	inputJSON, err := ioutil.ReadFile("testdata/exampleEnrichedContentModel.json")
+	assert.NoError(err, "Unexpected error")
+	input := strings.Replace(string(inputJSON), "FTCOM-METHODE", "invalid", 1)
```
Comment (khatton-ft) [inline service/message_handler_test.go]: We should look at whether this is correct, it doesn't seem obvious for a Spark test.

File: vendor/vendor.json [hunk 2]
```diff
@@ -3,178 +3,178 @@
 	"ignore": "test",
 	"package": [
 		{
-			"checksumSHA1": "TVyK4buHhPUkpD0Pt9M7Zo2UAlI=",
+			"checksumSHA1": "2dtgENfeO6SICvxjruwb9f1CzLY=",
```
Comment (khatton-ft) [inline vendor/vendor.json]: Should migrate to dep.

Comment (galiri) [reply vendor/vendor.json]: migration to dep has errors as some of the libraries are being fetched from other libraries vendor folders. this should go in as a separate pr I think and may well require changing libraries and therefore regression testing.

**Comment by khatton-ft** [comment]
:+1:
If migrating to dep causes issues, skip it for this PR. It's tech debt to be picked up by the primary team for this service.

**Pull request merged**